hack/ci-e2e.sh

```diff
@@ -54,11 +54,33 @@ docker exec vbmc vbmc list
 # These variables are used by the tests. They override variables in the config file.
 # This IP is defined by the network we created above.
 # Together with the VBMC_PORT this becomes the BMC_ADDRESS used by the BMH in the test.
-export BMC_ADDRESS="ipmi://192.168.222.1:${VBMC_PORT}"
+IP_ADDRESS="192.168.222.1"
+export BMC_ADDRESS="ipmi://${IP_ADDRESS}:${VBMC_PORT}"
 export BOOT_MAC_ADDRESS
 # These are the VBMC defaults (used since we did not specify anything else for `vbmc add`).
 export BMC_USER=admin
 export BMC_PASSWORD=password
+IMAGE_FILE="cirros-0.5.2-x86_64-disk.img"
+export IMAGE_MD5SUM="b874c39491a2377b8490f5f1e89761a4"
+export IMAGE_URL="http://${IP_ADDRESS}/${IMAGE_FILE}"
+
+## Setup image server
+# Create a directory for images
+mkdir -p images
+pushd images
+
+## Setup image server
+# Check if IMAGE_FILE already exists
+if [[ ! -f $IMAGE_FILE ]]; then
+    wget http://download.cirros-cloud.net/0.5.2/$IMAGE_FILE
+else
+    echo "$IMAGE_FILE already exists. Skipping download."
```

Comment:
Nit: Use brackets and quoting everywhere for consistency and safety, in case there would be special characters in the string.
Any reason why you chose 0.5.2 btw? I have no problem with it, just curious since I see 0.6.2 is the latest.
```suggestion
if [[ ! -f "${IMAGE_FILE}" ]]; then
    wget "https://download.cirros-cloud.net/0.5.2/${IMAGE_FILE}"
else
    echo "${IMAGE_FILE} already exists. Skipping download."
```